cs/src/Contracts/TunnelEndpoint.cs

```diff
@@ -69,6 +75,13 @@ public abstract class TunnelEndpoint
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public string? PortSshCommandFormat { get; set; }
 
+    /// <summary>
+    /// Gets or sets the Ssh command where the Ssh Gateway can connect to the default ssh port
```

Comment:
Should this reference "ssh client" rather than "ssh gateway"?

Reply:
yes, that is correct. Fixing it.